platform_versioned_docs/version-24.2/compute-envs/azure-batch.mdx

```diff
@@ -311,7 +433,7 @@ Create a manual Seqera Azure Batch compute environment:
     Configuration settings in this field override the same values in the pipeline repository `nextflow.config` file. See [Nextflow config file](../launch/advanced.mdx#nextflow-config-file) for more information on configuration priority. 
     :::
     :::info
-    To use managed identities, Platform requires Nextflow version 24.06.0-edge or later. Add `export NXF_VER=24.06.0-edge` to the **Global Nextflow config** field for your compute environment to use this Nextflow version by default. 
+    To use managed identities, Platform requires Nextflow version 24.06.0-edge or later.
```

Comment:
⛏️ Does "Platform" really require it? If Managed Identity is being leveraged by the head job in the Batch Pool, isn't this more a matter of "The job your launch via Tower into the Managed-Identity-Pool must use Nextflow 24.06.0-edge or greater"?